SCClassLibrary/Common/Control/Server.sc

```diff
@@ -1085,8 +1085,15 @@ Server {
 
 		addr.sendMsg("/quit");
 
-		if(watchShutDown and: { this.unresponsive }) {
-			"Server '%' was unresponsive. Quitting anyway.".format(name).postln;
+		if(this.unresponsive) {
+			if(pid.notNil) {
+				"Server '%' was unresponsive. Killing forcefully.".format(name).postln;
```

Comment:
As we have become more sensitive to human metaphors recently, a suggestion that has less police associations:

```
"Server '%' is currently unresponsive. Forcing process to stop via system command."
```